src/project.js

```diff
@@ -40,7 +40,7 @@ const altitudinalZoneList = types.altitudinalZone
 const getNextHeigtLevel = currentaltitudinalZone =>
   altitudinalZoneList[altitudinalZoneList.indexOf(currentaltitudinalZone) + 1];
 
-function projectionReducer(location) {
+function projectionReducer(location, targetaltitudinalZone) {
```

Comment:
```suggestion
function projectionReducer(location, targetAltitudinalZone) {
```

Please use camelCase.